04.class/class.js

```diff
@@ -0,0 +1,119 @@
+const fs = require('fs')
+const { MultiSelect } = require('enquirer')
+const minimist = require('minimist')(process.argv.slice(2))
+
+const reader = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+class Memo {
+  constructor () {
+    this.files = fs.readdirSync('./memo_data', 'utf-8')
+  }
+
+  buildChoice () {
+    return this.files.map((path) => {
+      const arrayOfContents = this.buildArrayOfContents(path)
+      const choice = { name: `${arrayOfContents[0]} in ${path}`, value: path }
+      return choice
```

Comment:
質問いただいた件ですが、僕だったらこう書きますね。

```suggestion
      return { name: `${arrayOfContents[0]} in ${path}`, value: path }
```

変数に入れてその値を即リターン、というのは基本的に変数に代入する意味がないと思っています（まれに例外はありますが）。

Reply:
@JunichiIto さんご回答していただきありがとうございます！

> 変数に入れてその値を即リターン、というのは基本的に変数に代入する意味がないと思っています（まれに例外はありますが）。

なるほどです！原則的に変数に入れ即リターンの場面では変数に入れる必要はないと考えておきます！

こちらのコミットで修正しましたのでご確認よろしくおねがいします。
[f081bc0](https://github.com/OdenTakashi/js-practices/pull/4/commits/f081bc0fb9ad54fa48661cc693458b1b9d053f50)